lib/options.js

```diff
@@ -124,10 +124,7 @@ function assignPublishing(opts) {
 
   if (opts.ably && opts.channel) {
     try {
-      opts.ably = opts.ably === true ? {} : opts.ably || {};
-      opts.ably.log = { level: 0 };
-
-      var ably = new require('ably').Rest(opts.ably);
+      var ably = new require('ably').Rest({log: {level: 0}});
```

Comment:
I like this much better :)
We can consider making this configurable when it becomes necessary.

Also, I think we should lock the version of Ably in `package.json`. At least the major version.

`"ably": "*"` 😨 

![image](https://cloud.githubusercontent.com/assets/639336/25675684/4081b33c-2ff4-11e7-9974-7cff208eae89.png)

`"ably": "^4.x.x"`
![image](https://cloud.githubusercontent.com/assets/639336/25675750/708205fa-2ff4-11e7-9be2-0f0361851142.png)

<https://semver.npmjs.com/>